content-src/experiments/notes.yaml

```diff
@@ -0,0 +1,79 @@
+id: 14
+title: 'Notes'
+slug: notes
+platforms: ['addon']
+thumbnail: /static/images/experiments/notes/experiments_experiment/thumbnail.png
+description: >
+  A simple notepad built into Firefox. Ever need to jot
+  something down while you browse the web? This is the experiment for you.
+introduction: >
+  Notes adds a simple, one-page notepad to your browser’s sidebar. We’ve added a few basic formatting options to start, but we want your feedback to tell us what features you’d like to see us add.
+image_twitter: /static/images/experiments/min-vid/social/notes-twitter.jpg
+image_facebook: /static/images/experiments/min-vid/social/notes-fb.jpg
+changelog_url: 'https://github.com/mozilla/notes/blob/master/CHANGELOG.md'
+contribute_url: 'https://github.com/mozilla/notes'
+bug_report_url: 'https://github.com/mozilla/notes/issues'
+discourse_url: 'https://discourse.mozilla-community.org/c/test-pilot/notes'
+privacy_notice_url: 'https://github.com/mozilla/notes/blob/master/docs/metrics.md'
+measurements:
+  - >
+    Notes will collect information about your use of the feature, such as how often you create notes
+    and how long you spend editing them, the number of characters you save in your notes, and whether
+    you use any advanced editing features (e.g., bold or stylized text).
+  - >
+    Notes will also collect information about your interaction with the add-on, such as how you
+    access the feature, and what buttons you may click.
+xpi_url: 'https://testpilot.firefox.com/files/@notes/latest'
```

Comment:
> We’ve added a few basic formatting options to start

> Notes will collect information about ...whether you use any advanced editing features (e.g., bold or stylized text).

We're calling the editing features both basic and advanced. We should just strike the word "advanced," right?